apps/server/src/modules/video-conference/video-conference.module.ts

```diff
@@ -1,32 +1,43 @@
 import { Module } from '@nestjs/common';
 import { HttpModule } from '@nestjs/axios';
 import { CalendarModule } from '@shared/infra/calendar';
-import { BBBService } from '@src/modules/video-conference/service/bbb.service';
 import { VideoConferenceRepo } from '@shared/repo/videoconference/video-conference.repo';
-import { VideoConferenceController } from '@src/modules/video-conference/controller/video-conference.controller';
 import { AuthorizationModule } from '@src/modules/authorization';
-import { VideoConferenceUc } from '@src/modules/video-conference/uc/video-conference.uc';
-import { CourseRepo, SchoolRepo, TeamsRepo, UserRepo } from '@shared/repo';
+import { CourseRepo, TeamsRepo, UserRepo } from '@shared/repo';
 import { SchoolModule } from '@src/modules/school/school.module';
-import { VideoConferenceResponseMapper } from '@src/modules/video-conference/mapper/vc-response.mapper';
 import { LoggerModule } from '@src/core/logger';
 import { ConverterUtil } from '@shared/common';
-import { SchoolService } from '../school';
+import { UserModule } from '@src/modules/user';
+import { BBBService, BbbSettings } from './bbb';
+import { VideoConferenceService } from './service';
+import { VideoConferenceDeprecatedUc } from './uc';
+import { VideoConferenceDeprecatedController } from './controller';
+import VideoConferenceConfiguration from './video-conference-config';
+import { VideoConferenceSettings } from './interface';
 
 @Module({
-	imports: [AuthorizationModule, CalendarModule, HttpModule, SchoolModule, LoggerModule],
-	controllers: [VideoConferenceController],
+	imports: [AuthorizationModule, CalendarModule, HttpModule, SchoolModule, LoggerModule, UserModule],
 	providers: [
-		VideoConferenceUc,
+		{
+			provide: VideoConferenceSettings,
+			useValue: VideoConferenceConfiguration.videoConference,
+		},
+		{
+			provide: BbbSettings,
+			useValue: VideoConferenceConfiguration.bbb,
+		},
 		BBBService,
-		SchoolRepo,
 		VideoConferenceRepo,
 		TeamsRepo,
 		CourseRepo,
 		UserRepo,
```

Comment:
Please remove Repos and use Import UserModule/LernroomModule and use UserService/CourseService